src/main/webapp/resources/js/pages/projects/settings/components/DeleteProject.jsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Checkbox, notification, Space, Typography } from "antd";
+import React from "react";
+import { deleteProject } from "../../../../apis/projects/project";
+
+/**
+ * React component to delete a project
+ * @param {number} projectId - identifier for the current project
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function DeleteProject({ projectId }) {
+  const [disabled, setDisabled] = React.useState(true);
+  const [loading, setLoading] = React.useState(false);
+
+  const doDeleteProject = () => {
+    setLoading(true);
+    deleteProject(projectId).catch((message) => {
+      setLoading(false);
+      notification.error({ message });
+    });
+  };
+
+  return (
+    <>
+      <Typography.Title level={2}>
+        {i18n("DeleteProject.title")}
+      </Typography.Title>
+
+      <Space direction="vertical" size={"middle"}>
+        <Typography.Text type="danger" underline>
```

Comment:
Can we change this to an alert or something? I think it would look better but I will leave that up to you :)

Reply:
Sure, why not.  Updated in 51f35e2